misc/python/materialize/checks/all_checks/optimizer_notices.py

```diff
@@ -55,7 +55,7 @@ def validate(self) -> Testdrive:
             dedent(
                 f"""
                 $postgres-execute connection=postgres://mz_system:materialize@${{testdrive.materialize-internal-sql-addr}}
-                ALTER SYSTEM SET enable_rbac_checks TO false
+                SET LOCAL enable_session_rbac_checks TO false
```

Comment:
Use `SET SESSION` in the **same** session, so with `> ` prefix, not with `$postgres-execute` (which creates a new session)